src/ert/parsing/context_values.py

```diff
@@ -0,0 +1,88 @@
+from typing import Union
+
+from .file_context_token import FileContextToken
+
+
+class ContextBool:
+    token: FileContextToken
+    keyword_token: FileContextToken
+
+    def __init__(self, val: bool, token: str, keyword_token: FileContextToken):
```

Comment:
Mypy complains about a mismatch of types in `self.token = token`

Reply:
Oops, should be FileContextToken, fixed